ragna/deploy/_api/orm.py

```diff
@@ -19,14 +19,16 @@ class Json(types.TypeDecorator):
 
     cache_ok = True
 
-    def process_bind_param(self, value: Any, dialect: Dialect) -> str:
+    def process_bind_param(self, value: Any, dialect: Dialect) -> Optional[str]:
+        if value is None:
+            return value
```

Comment:
We now also have a nullable JSON field with the `MetadataFilter` and thus also need to handle this here.